ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/linear_optimisation/fillers/MaxLoopFlowFiller.java

```diff
@@ -34,12 +37,14 @@
 public class MaxLoopFlowFiller implements ProblemFiller {
 
     private boolean isLoopFlowApproximation;
+    private double loopFlowAcceptableAugmentation;
     private double loopFlowConstraintAdjustmentCoefficient;
     private double loopFlowViolationCost;
     private SensitivityAnalysisParameters sensitivityAnalysisParameters;
 
-    public MaxLoopFlowFiller(boolean isLoopFlowApproximation, double loopFlowConstraintAdjustmentCoefficient, double loopFlowViolationCost, SensitivityAnalysisParameters sensitivityAnalysisParameters) {
+    public MaxLoopFlowFiller(boolean isLoopFlowApproximation, double loopFlowAcceptableAugmentation, double loopFlowConstraintAdjustmentCoefficient, double loopFlowViolationCost, SensitivityAnalysisParameters sensitivityAnalysisParameters) {
```

Comment:
The new parameter loopFlowAcceptableAugmentation does not seem to be used in that class. Could be removed from constructor